.github/workflows/docs.yml

```diff
@@ -37,3 +37,12 @@ jobs:
           branch: gh-pages # The branch the action should deploy to.
           folder: docs/out # The folder the action should deploy.
           clean: true # Automatically remove deleted files from the deploy branch
+
+      - name: Generate and Save Algolia Object
+        uses: actions/setup-node@v2
+      - run: yarn
+      - run: yarn run buildsearch
+        env:
+          CI: true
+          ALGOLIA_APPLICATION_ID: ${{secrets.ALGOLIA_APPLICATION_ID}}
```

Comment:
To make it consistent, please change this `ALGOLIA_APPLICATION_ID` to `NEXT_PUBLIC_ALGOLIA_APPLICATION_ID`